File: 240728/007/007.c
#include <stdio.h>


typedef struct secret {
    char code[10];
    char place[10];
    int time;
} secret;

int main() {
    // 여기에 코드를 작성해주세요.
    secret m;
    scanf("%s %s %d", m.code, m.place, &m.time);
    printf("secret code : %s\n", m.code);
    printf("meeting point : %s\n", m.place);
    printf("time : %d", m.time);
    return 0;
}

File: 240728/007/007.py
class secret:
    def __init__(self, code, place, time):
        self.code = code
        self.place = place
        self.time = time

s_code, s_place, time = tuple(input().split())

s = secret(s_code, s_place, int(time))

print(f"secret code : {s.code}")
print(f"meeting point : {s.place}")
print(f"time : {s.time}")